cmd/lastfm.go

```diff
@@ -88,7 +88,7 @@ func getAlbums(username string, period Period, count int, imageSize string) ([]A
 
 		// No albums to return
 		if len(lastFMResponse.TopAlbums.Album) == 0 && page == 1 {
-			return nil, errors.New("No Albums found! Is the username correct?")
+			return nil, errors.New("no Albums found! Is the username correct?")
```

Comment:
I need to figure out how to use errors in `go` properly. I feel like, as the linter is suggesting, that the error here shouldn't be the message that is returned to the end user.

Reply:
Going by https://google.github.io/styleguide/go/decisions.html#error-strings

>Error strings should not be capitalized (unless beginning with an exported name, a proper noun or an acronym) and should not end with punctuation. This is because error strings usually appear within other context before being printed to the user.

And some reasoning around it here https://stackoverflow.com/questions/68792696/why-error-messages-shouldnt-end-with-a-punctuation-mark-in-go